crates/ruff/src/rules/flake8_bandit/snapshots/ruff__rules__flake8_bandit__tests__S609_S609.py.snap

```diff
@@ -0,0 +1,41 @@
+---
+source: crates/ruff/src/rules/flake8_bandit/mod.rs
+---
+S609.py:4:1: S609 Possible wildcard injection in call
```

Comment:
I assume you copied this message from the upstream plugin? I struggle to understand the message. Which part of my code is triggering the `wildcard` injection? Unfortunately, I also don't know how to improve the message much without our Diagnostic's supporting an `advice` that is shown to explain the rule in more detail.

Reply:
Yes, the message comes from Bandit: https://bandit.readthedocs.io/en/latest/plugins/b609_linux_commands_wildcard_injection.html

I did not bother to include the called function name in the message, but I don't think doing so would make it any clearer. Anyway I don't think it's a particularly good one either. Something like "Potentially dangerous use of wildcard in shell call" could be better, but then again I'm not sure how close to upstream Ruff wants to stick.